jwst/extract_2d/tests/test_grisms.py

```diff
@@ -41,6 +41,8 @@
 # Default wcs information
 # This is set for a standard nircam image just as an example
 # It does not test the validity of the absolute results
+# for create_tso_wcsimage, set the width of the output image to this value:
+NIRCAM_TSO_WIDTH = 10
```

Comment:
Is this being reduced from 2048 to 10 just to save time/memory in the unit test?

Reply:
Yes.  Either the Travis or Jenkins CI test (I forget which) failed because it couldn't finish in the allotted 10 minutes.